app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -36,6 +36,7 @@ def prov_field_changed
           all_dialogs.each do |dialog_name, dialog|
             if dialog[:display] == :show && dialog_name == @edit[:new][:current_tab_key]
               page.replace_html(dialog_name, :partial => dialog_partial_for_workflow, :locals => {:wf => @edit[:wf], :dialog => dialog_name})
+              page << "$('[data-toggle=popover]').popovers()"
```

Comment:
The element is tagged with 'popover', but the initialization function is 'popovers' (because things must never be easy):

https://github.com/patternfly/patternfly/blob/master/src/js/patternfly.js#L38

Without the initialization, nothing happens when clicking upon the blue i.